### PR TITLE
Created initial figure set

### PR DESCRIPTION
**Pull request by Toms-R**
Created initial figure set - https://github.com/klepzers/chess/issues/2

### CODE REVIEW

**Review by klepzers — CHANGES_REQUESTED** [review]
visuals.js izmaiņas noņem nost no commita! :)

**Comment by klepzers** [comment]
git add . vairs neies cauri

File: visual.js [hunk 1]
```diff
@@ -70,16 +70,18 @@ function clickOnPiece(e) {
 
 function drawPiece(piece) {
     var img = new window.Image();
-    img.addEventListener("load", function () {
```
Comment (klepzers) [inline visual.js]: This change is not relevant to current PR

File: visual.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ function drawBoard() {
 }
 
 function drawInitialPieces() {
-    startingPositions.forEach(function (piece) {
+    startingPositions.forEach(function(piece) {
```
Comment (klepzers) [inline visual.js]: This change is not relevant to current PR

File: visual.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ canvas.height = chessBoardSize;
 let squareSize = chessBoardSize / squaresInRow;
 let totalSquares = Math.pow(squaresInRow, 2);
 
-// Define starting positions from logic.js.
+// Define starting positions from logic.js.
```
Comment (klepzers) [inline visual.js]: This change is not relevant to current PR

File: logic.js [hunk 4]
```diff
@@ -1,34 +1,197 @@
...
+        {
+            "isWhite": true,
+            "type": "R",
+            x: 8,
+            y: 8
+        }
 
     ]
-}
+}
```
Comment (klepzers) [inline logic.js]: No newline at the end of the file.

**Review by klepzers — CHANGES_REQUESTED** [review]

**Review by klepzers — APPROVED** [review]
Everything is fine now :)

**Review by klepzers — APPROVED** [review]
console.log should be removed after yellow dots functionality is done.

**Review by austramarta — APPROVED** [review]

**Pull request merged**